tests/bluetooth/tester/src/gap.c

```diff
@@ -890,6 +906,9 @@ void tester_handle_gap(uint8_t opcode, uint8_t index, uint8_t *data,
 	case GAP_SET_MITM:
 		set_mitm(data, len);
 		return;
+	case GAP_OOB_SC_GET_LOCAL_DATA:
```

Comment:
Add this functionality to GAP supported_commands in line 133 as well. It will enable get_oob_sc_local_data func to be found with gap_read_supported_commands_rp

Reply:
Good catch